crates/mipsy_web/src/components/registers.rs

```diff
@@ -9,6 +9,10 @@ pub struct RegisterProps {
     pub tab: UseStateHandle<RegisterTab>,
 }
 
+const K0_INDEX: usize = 26;
```

Comment:
Can we not use register numbers from mipsy_lib?

Reply:
didn't even know that existed... If only I had intellisense working on windows : ^)